src/main/scala/viper/gobra/translator/transformers/hyper/SIFLowGuardTransformer.scala

```diff
@@ -482,11 +482,18 @@ trait SIFLowGuardTransformer {
     else Seqn(Seq.empty, Seq.empty)(s.pos, s.info, s.errT)
   }
 
-  def removeLow[N <: Node](n: N): N = {
-    n.transform{
-      case n: SIFLowExp => TrueLit()(n.pos, n.info, n.errT)
-      case n: SIFLowEventExp => TrueLit()(n.pos, n.info, n.errT)
+  def removeLow[N <: Node](n: N): Either[Seq[AbstractError], N] = {
+    var errs: Seq[AbstractError] = Seq.empty
+    val transformedN = n.transform{
+      case n: SIFLowExp =>
+        errs = errs :+ ConsistencyError(s"Low expression found: ${n.exp}", n.pos)
```

Comment:
I'm not sure if this is a good solution because every package that imports `sync` to use a Mutex will use the new spec for this function, which contains low annotations. Thus, this solution forces us to always enable SIF when verifying concurrent code

Reply:
I've also realized this after looking at the failing testcases ^^
Maybe we should distinguish between imported and non-imported SIF expressions and ignore the former while cause errors for the latter (if SIF is disabled).
What I want to avoid is any confusion as in my program contains SIF expressions but I (e.g. by mistake) configured Gobra not to do a product construction and, thus, I'm thinking I've verified SIF but actually Gobra didn't

Reply:
I've just changed the implementation to report an error only if the resulting proof obligations will be different. In particular, if an abstract method or function has a low expression (because it's abstract or imported) then we just replace the expression by `true` and do not report an error